M2/Macaulay2/e/BasicPolyList.hpp

```diff
@@ -25,16 +25,17 @@ long bytesUsed(const BasicPolyList& F);
 class BasicPolyListStreamCollector
 {
 public:
-  using Coefficient = int32_t;
+  using Coefficient = mpz_class;
   using VarIndex = int32_t; // TODO: these must match BasicPoly above.
   using Exponent = int32_t;
   using Component = int32_t;
+  using ModulusType = int32_t;
```

Comment:
What characteristics do we want to use here?  mathic uses roughly 16 bit primes, while msolve seems to want much larger primes.  Are they still less than e.g. 2^31?